CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
-## 0.8.0 - 2018-05-01
+## 0.8.0 - 2018-05-03
```

Comment:
This means people won't get updated to the new extension version until they've updated VS Code (which people often put off when they're working on stuff).

Reply:
I agree with you, but I don't feel strongly about it. Chris and Matt don't seem to care. Feel free to add your opinion to the discussion here: https://github.com/Microsoft/vscode-azuretools/issues/167